### PR TITLE
Add Wolt

### PR DESCRIPTION
**Pull request by jtagcat**

### CODE REVIEW

**Comment by jtagcat** [comment]
Fixed conflict.

File: _data/food.yml [hunk 1]
```diff
@@ -34,10 +34,18 @@ websites:
...
+      facebook: woltapp
+      email_address: info@wolt.com
  
     - name: Zomato
       url: https://www.zomato.com
       img: zomato.png
       tfa: No
       twitter: Zomato
-      facebook: zomato
+      facebook: zomato
```
Comment (Carlgo11) [inline _data/food.yml]: All files should end with a blank line.
I recommend you to use a decent text editor like Atom, Notepad++, VS Code, Sublime etc. They fix that for you.
```suggestion
      facebook: zomato

```

File: _data/food.yml [hunk 2]
```diff
@@ -34,10 +34,18 @@ websites:
...
       twitter: tacobell
       facebook: tacobell
+  
+    - name: Wolt
+      url: https://wolt.com/
+      img: wolt.png
+      tfa: No
+      twitter: woltapp
+      facebook: woltapp
+      email_address: info@wolt.com
```
Comment (Carlgo11) [inline _data/food.yml]: info@ seems to be a general inquiry address, support@ would probably fit better.  _(source: [wolt.com/help#contact-support](https://wolt.com/help#contact-support))_ 
```suggestion
      email_address: support@wolt.com
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
1.5Kb logo version:
wolt

**Comment by jtagcat** [comment]
Changes made.

**Comment by Carlgo11** [comment]
@RasmusKallas you forgot the logo change 😉 
Also, when do you sleep? 👀 
Should be around 01:30am in Estonia now.

**Review by Carlgo11 — APPROVED** [review]

**Comment by jtagcat** [comment]
I don't :)
Usually 2 hours a day and whenever I get near a bed the next day (same day), then 14 hours and repeat.. 5am, 6pm, 7am, 4pm, 2am, 5am, 4am.. The Sunday->Monday seems the hardest
I just collapse on the weekends ;) Very healthy.....

**Comment by Carlgo11** [comment]
I guess that's one way to manage sleep. To simply not do it. ¯\_(ツ)_/¯

**Comment by jtagcat** [comment]
It's not my fault I need to be sleeping elsewhere by 8:15, When I get to sleep normal for 3-5 days, I usually wake 10-11, I'd recommend 11am for a start.. Also I don't like git really.. It seems overly complicated, while managing multiple repositories and branches at once, tough I can say it from only github's point, git's setup has scared me away, even tough I do well with CLI..

**Comment by jtagcat** [comment]
Waait, Sweedeen, isn't it :46? @Carlgo11

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**